src/pdf.rs

```diff
@@ -5,53 +5,59 @@ use std::{
 };
 
 use image::DynamicImage;
-
+use once_cell::sync::OnceCell;
 use pdfium_render::prelude::*;
 
+static PDFIUM: OnceCell<Pdfium> = OnceCell::new(); // static initializers must impl Sync + Send
```

Comment:
Todo points must be resolved before merging.

Reply:
Oh, I didn't realize that `OnceCell` implements `Sync + Send`. Sorry, wrong comment :)